src/StatusQ/Popups/StatusSearchPopup.qml

```diff
@@ -0,0 +1,198 @@
+                    input.icon.name: "search"
+                    input.placeholderText: "Search"
+                    input.bgVisible: false
+                }
+            }
+            StatusMenuSeparator { Layout.fillWidth: true }
+            Item {
+                Layout.preferredWidth: parent.width
+                Layout.preferredHeight: 58
+                StatusButton {
+                    id: searchOptionsMenuButton
+                    anchors.left: parent.left
+                    anchors.leftMargin: 16
+                    anchors.verticalCenter: parent.verticalCenter
+                    implicitWidth: (contentItemRowLayout.width + 8)
+                    implicitHeight: 32
+
+                    contentItem: Item {
+                        anchors.fill: parent
+                        RowLayout {
+                            id: contentItemRowLayout
+                            anchors.centerIn: parent
+                            spacing: 2
+                            StatusBaseText {
+                                text: "In: "
+                            }
+                            StatusRoundedImage {
+                                implicitWidth: 16
+                                implicitHeight: 16
+                                visible: !!root.selectionBadgeImage
+                                image.source: root.selectionBadgeImage
+                            }
+                            StatusLetterIdenticon {
+                                implicitWidth: 16
+                                implicitHeight: 16
+                                letterSize: 11
+                                visible: ((root.selectionBadgeImage === "") && (root.selectionBadgePrimaryText !== "Anywhere"))
+                                color: root.selectionBadgeIdenticonColor
+                                name: root.selectionBadgePrimaryText
+                            }
+                            StatusBaseText {
+                                text: root.selectionBadgePrimaryText
+                            }
+                            StatusIcon {
+                                Layout.alignment: Qt.AlignVCenter
+                                visible: !!root.selectionBadgeSecondaryText
+                                color: Theme.palette.baseColor1
+                                icon: "next"
+                            }
+                            StatusBaseText {
+                                visible: !!root.selectionBadgeSecondaryText
+                                text: "#" + root.selectionBadgeSecondaryText
+                            }
+                            StatusIcon {
+                                Layout.alignment: Qt.AlignVCenter
+                                icon: "chevron-down"
+                            }
+                        }
+                    }
+                    onClicked: { searchPopupMenuLoader.item.popup(); }
+                }
```

Comment:
As seen in previous screenshots attached in this review, the background of this button doesn't match the design. This needs update as well

Reply:
Done, updated. Also switched StatusBaseButton as it was (Rectangle) and made this one a QtQuick Controls 2 Button instead